app/models/importers/number_of_views_by_organisation.rb

```diff
@@ -0,0 +1,16 @@
+module Importers
+  class NumberOfViewsByOrganisation
+    def run(slug)
+      organisation = ::Organisation.find_by(slug: slug)
+
+      content_items = ::ContentItem.where(organisation_id: organisation.id)
+      base_paths = content_items.pluck(:base_path)
+
+      results = GoogleAnalyticsService.new.page_views(base_paths)
```

Comment:
I have created a ticket so we explore pagination here. We could have two type of issues with big organisations:

1. We `pluck` too many items from the database so we could have memory issues. I would advise using [find_each][1]
2. We need to investigate what is the limit in Google Analytics as, it is very likely, that there is an upper limit for the response.

No need to take an action now. Just pointing it out for the incoming ticket.

[1]: http://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each